src/Nethermind/Nethermind.Blockchain/Receipts/NullReceiptStorage.cs

```diff
@@ -44,10 +44,10 @@ public bool TryGetReceiptsIterator(long blockNumber, Keccak blockHash, out Recei
 
         public long? LowestInsertedReceiptBlockNumber
         {
-            get => long.MaxValue;
+            get => 1;
```

Comment:
I do not think we should have any logic inside the null storage.
1 has a very specific magical meaning. We should either return null here or long.maxValue.

Reply:
But I think it is me who is at fault of making 1 magical.
Can we maybe extract it as a constant somewhere? AllReceiptsDownloaded =1